lib/XTR/WidgetCompatibilityTask.php

```diff
@@ -139,7 +109,115 @@ public function run()
             $this->appendRunMessage('Disabling widget compatibility task.');
             $this->log->debug('Disabling widget compatibility task.');
         }
-        $this->log->info('Total widgets upgraded: '. $countWidgets);
-        $this->appendRunMessage('Total widgets upgraded: '. $countWidgets);
+
+    }
+
+    /**
+     *
+     * @param string $type
+     * @param int $version
+     * @return false|\PDOStatement
+     */
+    private function executeStatement(string $type, int $version): bool|\PDOStatement
+    {
+        $sql = '
+          SELECT widget.widgetId,
```

Comment:
I can see you're using `widgetId` - but what are you using the other columns for?